src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
+  </a>
```

Comment:
Use destructuring for getting access to values of props object
```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => (
  <a className="UserInfo" href={`mailto:${email}`}>
    {name}
  </a>
```

you render user only if it exists